vantage/business_metric_resource_test.go

```diff
@@ -122,3 +122,136 @@ func testAccVantageBusinessMetricTf_basic(id string, title string, valuesStr str
 		`, id, title, valuesStr,
 	)
 }
+
+func TestAccBusinessMetric_cloudwatch(t *testing.T) {
+	t.Skip("Skipping until we have support for Integrations/AccessCredentials")
+	// NOTE: You will also have to replace the hard coded access credential token in the test data.
```

Comment:
I've been testing localhost, but won't be able to support these tests until the above mentioned is fulfilled. @whereandy Should I log a ticket for engineering team?